packages/react/tsconfig.build.json

```diff
@@ -29,7 +29,7 @@
     "baseUrl": "./",
     "rootDir": "./src",
     "paths": {
-      "@clr/core/*": ["../../dist/core/*"]
+      "@clr/core/*": ["../core/dist/core/*"]
```

Comment:
Something to double check here is make sure that both versions are the same. I initially pointed it to the parent-level `dist` folder because that's what gets published to npm.

Reply:
Reverted this change. I think once we finish setting up the yarn workspaces we wont need TS path mapping